### DISCUSSION TITLE
Use Renovate instead of Dependabot?

### DISCUSSION THREAD

**Discussion by JoergSiebahn** [SDA-SE/sda-dropwizard-commons#1373] [Ideas]
I stumbled up on [this PR](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1339). If Renovate is now an approved tooling in the SDA it could [save us some time](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1339#issuecomment-1003924470) compared to Dependabot.

**Comment by christopher-cudennec** [comment]
The answer from the platform meeting was: Yes we can do it. And since it saves us time we should probably prefer Renovate over Dependabot in sda-dropwizard-commons.
